lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +44,23 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str):
+            return None
+        self.text = text
```

Comment:
1. Creating new attributes somewhere other than `__init__` is a bad practice! Can you explain why?
2. Why do you need this attribute anyway?

Reply:
Fixed